checkbox-ng/checkbox_ng/launcher/subcommands.py

```diff
@@ -343,6 +343,7 @@ def _start_new_session(self):
         if self.launcher.app_version:
             title += ' {}'.format(self.launcher.app_version)
         runner_kwargs = {
+            'dry_run' : self.launcher.dry_run,
```

Comment:
if it's a part of configs/launchers then this doesn't have to be pushed to the runner like this. This runner_kwargs is for callables that need to cross the RPyC boundry.